msg/MotionSequenceItem.msg

```diff
@@ -0,0 +1,5 @@
+# The plan request
+MotionPlanRequest req
+
+# The blend radius in meter (used between this and the next command), 0 means no blending
```

Comment:
This is pretty algorithm-specific. Can you add more context for how the blend radius might be used?

Reply:
please see my latest commit